content/blog/posts/o3de-introducing-design-system.md

```diff
@@ -0,0 +1,95 @@
+
+![Preview @mdash; for the O3DE UI component guidelines](/images/blog/intro-to-bjds/ui-components-preview.jpg)
+
+#### UX Patterns
+
+UX Patterns are a series of best practice solutions that a designer or developer should follow when creating new workflows and components in the editor. Unlike UI Components, where it's mostly about the visual aesthetics and how it mechanically operates, UX Patterns is more about how a user interacts and behaves for a task with a goal in mind.
+
+For e.g., the anatomy of a component card and it's different types can be found on the [UI Component > Card page](https://o3de.org/docs/tools-ui/component-library/uidev-card-component/), while the what contents go in the card, how a user interacts with it and how O3DE interacts with the card itself can be found on the [UX Patterns > Component Card UX Patterns](https://o3de.org/docs/tools-ui/ux-patterns/component-card/overview/) page.
+
+![Preview image for the O3DE component card guidelines](/images/blog/intro-to-bjds/componentcard-ux-preview.jpg)
+
+Currently we have two UX Pattern pages on O3DE Learn: Component Card Properties and Writing UI Message guidelines. We plan to roll out more pages in the future coming months, so stay tuned.
+
+#### O3DE Branding Guidelines
+
+Our [brand guidelines section](https://o3de.org/docs/tools-ui/branding-guidelines/) includes information on how to apply the O3DE brand, and downloadable assets such as; a PDF style guide, logos files and fonts etc. These pages are especially important if you are looking to represent the O3DE logo in your product or marketing website. 
+
+![Preview image for the O3DE branding guidelines](/images/blog/intro-to-bjds/branding-guidelines-preview.jpg)
+
+If there's an asset that's missing that you're looking for, don't hesitate to [file a ticket](https://github.com/o3de/sig-ui-ux/issues/new/choose) and the community will help get the right asset you need. We are always happy to help!
+### BlueJay Design System's Roadmap
+
+![Diagram showing the tenets of the O3DE design system](/images/blog/intro-to-bjds/roadmap2.jpg)
+
+Above is our planned roadmap for 2021–2022. These timeframes are estimates based off our current SIG-UI-UX contributions rate. Times may vary; if we are able to gather more community support and help, items could be completed even sooner.
+
+For a full detailed breakdown on our roadmap, you can read our article here.
+
+#### Looking Forward
+
+The next steps for BJDS involve planning out resources and commitments towards achieving these goals. As Phase 2 comes to a close soon, Phase 3 is will be achievable with the help of the SIG-UI-UX community effort as a whole. As a core contributor towards this design system, I will be looking to help kick start some of these new pages to get the ball going. 
```

Comment:
- "kick start" and "get the ball going" is redundant. I suggested one option below: "I am looking to help kick-start some of these new pages." Another option can be "I'm looking to help some of these new pages to get the ball going". 
```suggestion
The next steps for BJDS involve planning out resources and commitments towards achieving these goals. As Phase 2 comes to a close soon, Phase 3 will be achievable with the help of the SIG-UI-UX community effort. As a core contributor towards this design system, I am looking to help kick-start some of these new pages. 
```